frontend/src/shared/ui/Button/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+
+interface ButtonProps {
+	content: string;
+	onClick: () => void;
+}
+
+export function Button({ content, onClick }: ButtonProps) {
+	return <StyledButton onClick={onClick}>{content}</StyledButton>;
+}
+
+const StyledButton = styled.button`
```

Comment:
```suggestion
export const Button = styled.button`
```
This would be simpler and more extensible